public/static/docs/command-reference/import.md

```diff
@@ -15,20 +15,21 @@ import.
 usage: dvc import [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
-  path        Path to data within DVC repository.
+  url         Location of DVC project or Git repository to download from
+  path        Path to a file or directory within the project or repository
+
```

Comment:
Minor: there's still an extra blank line here.

Reply:
Now obliterated!

Reply:
Destroyed that line 💥